spec/spec.md

```diff
@@ -340,149 +368,126 @@ where:
 {
     “auth_token”: “<auth_token>”,
     “accounts”: [
-        {“address”: “<address>", “label”: “<label>”},
+        {
+            “address”: “<address>", 
+            “label”: “<label>”, 
+            "chains": ["<chain_id>", ...], 
```

Comment:
How would a multi-chain account function in the remainder of this spec? The other methods don't include a chain specifier, so there wouldn't be any way to use an alternate chain.

Reply:
Yeah so my thought here is this: the chains in the account metadata are only there to tell a dapp what chains an account can be used on. The dapp would still need to obtain an `auth_token` for the account(s) and chain they want to use. 

Now in practice, this does seem unnecessary/not useful given that a dapp cannot specify which account is authorized, and only one chain is allowed per `auth_token`. So at this time, this field is really just aligning with the wallet standard WalletAccount interface.